filebeat/cmd/run.go

```diff
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/beats/filebeat/beater"
+	"github.com/elastic/beats/libbeat/beat"
+)
+
+// RunCmd runs the beat
+var RunCmd = &cobra.Command{
```

Comment:
Any reason this is not also part of libbeat? Is there anything beat specific about `RunCmd`?